CRM/Contribute/BAO/Contribution/Utils.php

```diff
@@ -475,7 +475,8 @@ public static function getFirstLastDetails($contactID) {
    */
   public static function calculateTaxAmount($amount, $taxRate) {
     $taxAmount = array();
-    $taxAmount['tax_amount'] = round(($taxRate / 100) * CRM_Utils_Rule::cleanMoney($amount), 2);
```

Comment:
Fundamentally wrong to round here - as at this stage - in this basic Utils function quantity has not yet been taken into account. Rounding here causes incorrect Sales Tax calculations.